### PR TITLE
Update outdated link to docker instructions

### PR DESCRIPTION
**Pull request by JackCaoG**
issue being brought up in https://github.com/pytorch/xla/issues/4393#issuecomment-1364800733

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ If a single TPU VM does not suit your requirment, you can consider using TPU Pod
 
 
 ## Available images and wheels
-The following pre-built docker images are available to run on Cloud TPU VMs (see [docker images](#DockerImage) for instructions):
+The following pre-built docker images are available to run on Cloud TPU VMs (see [this doc](https://cloud.google.com/tpu/docs/pytorch-xla-ug-tpu-vm#docker-tpuvm) for TPUVM and [this doc](https://github.com/pytorch/xla/blob/master/docs/gpu.md#docker) for GPU):
```
Comment (will-cromar) [inline README.md:62]: `... available to run on Cloud TPU VMs (see this doc) for TPU VM and ...` is redundant. Can you remove one of the references to "TPU VM"?

Comment (JackCaoG) [reply README.md]: done

**Comment by JackCaoG** [comment]
@will-cromar not sure if making it a table is a good idea.. let me know what you thinks.

**Review by will-cromar — APPROVED** [review]
The table LGTM

**Pull request merged**